lms/djangoapps/email_marketing/migrations/0003_auto_20160714_0952.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('email_marketing', '0002_auto_20160623_1656'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='emailmarketingconfiguration',
+            name='sailthru_abandoned_cart_delay',
+        ),
+        migrations.RemoveField(
+            model_name='emailmarketingconfiguration',
+            name='sailthru_abandoned_cart_template',
+        ),
+        migrations.RemoveField(
+            model_name='emailmarketingconfiguration',
+            name='sailthru_purchase_template',
+        ),
+        migrations.RemoveField(
```

Comment:
Field removal is tricky for code that has already been deployed. Because of the way we deploy our servers, the old version of the server will be running when the new migration runs, so if you remove a field, the old version of the code will choke when it loads the model.